lib/chewy/rspec/update_index.rb

```diff
@@ -250,4 +244,17 @@ def array_difference(first, second)
     end
     difference.none?
   end
+
+  class Collector
+    attr_reader :updates
+
+    def initialize
+      @updates = []
+    end
+
+    def perform(body)
```

Comment:
Could you leave it as a comment there? because it is not clear from the declaration